js/aax.js

```diff
@@ -2037,6 +2039,214 @@ module.exports = class aax extends Exchange {
+        //        "realisedPnl": "-0.07792179",
+        //        "riskLimit": "10000000",
+        //        "riskyPrice": "3272.25",
+        //        "settleType": "VANILLA",
+        //        "stopLossPrice": "0",
+        //        "stopLossSource": 1,
+        //        "symbol": "ETHUSDTFP",
+        //        "takeProfitPrice": "0",
+        //        "takeProfitSource": 1,
+        //        "unrealisedPnl": "0.51620000",
+        //        "userID": "3829384",
+        //        "ts": 1641027194500
+        //    }
+        //
+        const contract = this.safeString (position, 'symbol');
+        market = this.safeMarket (contract, market);
+        const size = this.safeString (position, 'currentQty');
+        let side = undefined;
+        if (Precise.stringGt (size, '0')) {
+            side = 'buy';
+        } else if (Precise.stringLt (size, '0')) {
+            side = 'sell';
+        }
+        // const maintenanceRate = this.safeString (position, 'maintenance_rate');
+        // const notional = this.safeString (position, 'value');
+        const leverage = this.safeString (position, 'leverage');
+        const unrealisedPnl = this.safeString (position, 'unrealisedPnl');
+        const realisedPnl = this.safeString (position, 'realisedPnl');
+        // Initial Position Margin = ( Position Value / Leverage ) + Close Position Fee
+        const takerFee = this.fees.trading.taker;
+        // const feePaid = Precise.stringMul (takerFee, notional);
+        const initialMarginString = this.safeString (position, 'posMargin');
+        const percentage = Precise.stringMul (Precise.stringDiv (unrealisedPnl, initialMarginString), '100');
+        const timestamp = this.safeInteger (position, 'ts');
+        return {
+            'info': position,
+            'symbol': this.safeString (market, 'symbol'),
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'initialMargin': this.parseNumber (initialMarginString),
+            // 'initialMarginPercentage': this.parseNumber (Precise.stringDiv (initialMarginString, notional)),
+            // 'maintenanceMargin': this.parseNumber (Precise.stringMul (maintenanceRate, notional)),
+            // 'maintenanceMarginPercentage': this.parseNumber (maintenanceRate),
+            'entryPrice': this.safeNumber (position, 'avgEntryPrice'),
+            // 'notional': this.parseNumber (notional),
+            'leverage': parseInt (leverage),
+            'unrealizedPnl': this.parseNumber (unrealisedPnl),
+            'contracts': this.parseNumber (size),
+            'contractSize': this.safeNumber (market, 'contractSize'),
+            //     realisedPnl: position['realised_pnl'],
+            // 'marginRatio': undefined,
+            'liquidationPrice': this.safeNumber (position, 'liquidationPrice'),
+            'markPrice': this.safeNumber (position, 'marketPrice'),
+            // 'collateral': this.safeNumber (position, 'margin'),
```

Comment:
collateral is posMargin

Reply:
i have to check, but i'm pretty sure

Reply:
Whats the difference between notional and collateral?